README.md

```diff
@@ -151,6 +151,25 @@ bash build-and-publish-docker-image.sh --image <repo>/<imagename>:<imagetag>
 ```
 > Note: format of the imagetag will be `ci<release><MMDDYYYY>`. possible values for release are test, dev, preview, dogfood, prod etc.
 
+You can also build and push images for multiple architectures. This is powered by docker buildx
+```
+cd ~/Docker-Provider/kubernetes/linux/dockerbuild
+sudo docker login # if you want to publish the image to acr then login to acr via `docker login <acr-name>`
+# build and publish using docker buildx
+bash build-and-publish-docker-image.sh --image <repo>/<imagename>:<imagetag> --multiarch
+```
+
+or directly use the docker buildx commands
```

Comment:
is there any reason we should build the images separately and directly with buildx?

Reply:
i wanted to keep both options alive
1. existing where build and docker build are seperate
2.  new single dockerfile based 

additionally buildx makes it difficult to store images therefore it is better to have the previous option as well

From https://docs.docker.com/engine/reference/commandline/buildx_build/

"Currently, multi-platform images cannot be exported with the docker export type. The most common usecase for multi-platform images is to directly push to a [registry](https://docs.docker.com/engine/reference/commandline/buildx_build/#registry) (see registry)."